src/iotjs_handlewrap.h

```diff
@@ -41,31 +46,36 @@ namespace iotjs {
 // Handle closed, release handle, decrease ref count.
 //   ->
 // The javascript object now can be reclaimed by GC.
-class HandleWrap : public JObjectWrap {
- public:
-  HandleWrap(const iotjs_jval_t* jobject, uv_handle_t* handle);
-  // jobject: Object that connect with the uv handle
 
-  virtual ~HandleWrap();
+typedef struct {
+  iotjs_jobjectwrap_t jobjectwrap;
+  uv_handle_t* handle;
+  OnCloseHandler on_close_cb;
+} IOTJS_VALIDATED_STRUCT(iotjs_handlewrap_t);
```

Comment:
I looked up the definition of IOTJS_VALIDATED_STRUCT, and I found 
`typedef struct iotjs_classname_t { \  unsigned flag_create; \` in iotjs_def.h. 
`flag_create` is assignd to either IOTJS_VALID_MAGIC_SEQUENCE or IOTJS_INVALID_MAGIC_SEQUENCE, which is defined as 0xfee1c001, 0xfee1badd respectively. What about chainging `unsigned` to 32bit data type or poitner type, or making the magic number 16 bit?